File: app/src/main/java/com/frank/ffmpeg/model/AudioBean.java
package com.frank.ffmpeg.model;

/**
 * the model of audio data
 * Created by frank on 2020/1/7.
 */
public class AudioBean {

    //"codec_tag_string": "mp4a"
    private String audioCodec;

    //"sample_rate": "44100"
    private int sampleRate;

    //"channels": 2
    private int channels;

    //"channel_layout": "stereo"
    private String channelLayout;

    public String getAudioCodec() {
        if ("[0][0][0][0]".equals(audioCodec)) {
            return null;
        }
        return audioCodec;
    }

    public void setAudioCodec(String audioCodec) {
        this.audioCodec = audioCodec;
    }

    public int getSampleRate() {
        return sampleRate;
    }

    public void setSampleRate(int sampleRate) {
        this.sampleRate = sampleRate;
    }

    public int getChannels() {
        return channels;
    }

    public void setChannels(int channels) {
        this.channels = channels;
    }

    public String getChannelLayout() {
        return channelLayout;
    }

    public void setChannelLayout(String channelLayout) {
        this.channelLayout = channelLayout;
    }

}

File: app/src/main/java/com/frank/ffmpeg/model/AudioBean.kt
package com.frank.ffmpeg.model

/**
 * the model of audio data
 * Created by frank on 2020/1/7.
 */
class AudioBean {

    //"codec_tag_string": "mp4a"
    var audioCodec: String? = null
        get() = if ("[0][0][0][0]" == field) {
            null
        } else field

    //"sample_rate": "44100"
    var sampleRate: Int = 0

    //"channels": 2
    var channels: Int = 0

    //"channel_layout": "stereo"
    var channelLayout: String? = null

    var title: String? = null

    var artist: String? = null

    var album: String? = null

    var albumArtist: String? = null

    var composer: String? = null

    var genre: String? = null

    var lyrics: List<String>? = null

    var lrcLineList: List<LrcLine>? = null

}
